.github/workflows/generate-manifest.yml

```
@@ -0,0 +1,81 @@
+          REPO_URL="${{ github.event.inputs.repo_url }}"
+          REPO_NAME=$(echo "$REPO_URL" | sed 's/.*github\.com[:/]//' | sed 's/\.git$//' | tr '/' '-')
+          echo "repo_name=$REPO_NAME" >> $GITHUB_OUTPUT
+          echo "branch_name=add-manifest-$REPO_NAME" >> $GITHUB_OUTPUT
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: |
+            feat: add manifest for ${{ steps.repo-info.outputs.repo_name }}
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)